docs/03-apps-and-frameworks.md

```diff
@@ -0,0 +1,18 @@
+
+### Laravel
+  * [Account](http://account.stjude.cloud) / [Github](https://github.com/stjudecloud/account)
+  * [Viz Community](https://viz.stjude.cloud) / [Github](https://github.com/stjude/viz)
+  * [PeCan](https://pecan.stjude.cloud) / [Github](https://github.com/stjude/pecan) / `Uses Vue for Front-End`
+  * Vanguard / [Github](https://github.com/stjudecloud/vanguard) / `In Development`
+  * Intro to Genomics / [Github](https://github.com/stjude/introduction-to-genomics-for-engineers) / `In Development`
```

Comment:
Intro to Genomics should be under the React header and should have `Gatsby` added to the end